Sources/Mavsdk/Drone.swift

```diff
@@ -4,12 +4,13 @@ import MavsdkServer
 
 public class Drone {
     private let scheduler: SchedulerType
-    private var mavsdkServer: MavsdkServer?
+    public var mavsdkServer: MavsdkServer?
 
     public var action: Action!
     public var calibration: Calibration!
     public var camera: Camera!
     public var core: Core!
+    public var failure: Failure!
```

Comment:
@JonasVautherin Do we need to utilize this drone component, or is it specifically intended for injecting failures for testing and debugging?

Reply:
It is for testing and debugging. It is exposed because it is a plugin, but production apps should probably not use it :sweat_smile:.